Src/FluentAssertions/Common/StringExtensions.cs

```diff
@@ -78,11 +72,6 @@ public static string Combine(this string @this, string other, string separator =
             return other.Length != 0 ? other : string.Empty;
         }
 
-        if (other.Length == 0)
-        {
-            return @this;
-        }
```

Comment:
I introduced that in https://github.com/fluentassertions/fluentassertions/pull/817 but it might no longer be relevant?

Reply:
It wasn't covered by any code, so I decided to remove it.